main_column.py

```diff
@@ -1,164 +1,356 @@
-"""Distillation column model for 2018 PSE conference"""
+"""
+main_column.py
+Distillation column solution for 2018 PSE conference [1]
 
+This file imports the build_column function from gdp_column.py. gdp_column.py formulates the distillation column problem as a GDP problem.
+The code also imports the various functions from gdp.dsda.dsda_functions module.
+The main_column.py solves the Generalized Disjunctive Programming (GDP) using a variety of methods with GAMS solvers that are on the script.
+The methods used are Mixed Integer Non-Linear Programming (MINLP) reformulations and  GDP algorithms, and Logic-based Discrete Steepest Descent Algorithm(L-DSDA).
+The results are written to a CSV files.
+
+
+References:
+[1] Bernal, David E., et al. "Process Superstructure Optimization through Discrete Steepest Descent Optimization: a GDP Analysis and Applications in Process Intensification." Computer Aided Chemical Engineering. Vol. 49. Elsevier, 2022. 1279-1284.
+[2] Ghouse, Jaffer H., et al. "A comparative study between GDP and NLP formulations for conceptual design of distillation columns." Computer Aided Chemical Engineering. Vol. 44. Elsevier, 2018. 865-870.
+"""
+
+# List of differences when executing code in Albert's computer
```

Comment:
Add the date to this and computer characteristics

Reply:
Added Computer Specification, I do not have the time but I have the day.